client/src/component/sos/audit/RiskLimitForm.tsx

```diff
@@ -55,38 +55,37 @@ class RiskLimitForm extends React.Component<FormProps & any, FormState> {
 
         this.props.forms.riskLimit = this.state;
 
-        return (
-            <div>
-                <div>
-                    <label>
-                        Ballot Polling Audits (%)
-                        <NumericInput
-                            allowNumericCharactersOnly={ true }
-                            min={ toPercent(MIN_RISK_LIMIT) }
-                            max={ toPercent(MAX_RISK_LIMIT) }
-                            minorStepSize={ toPercent(0.001) }
-                            onBlur={ this.onBlur }
-                            stepSize={ toPercent(0.01) }
-                            value={ ballotPollingField }
-                            onValueChange={ this.onBallotPollingValueChange } />
-                    </label>
-                </div>
-                <div>
-                    <label>
-                        Comparison Audits (%)
-                        <NumericInput
-                            allowNumericCharactersOnly={ true }
-                            min={ toPercent(MIN_RISK_LIMIT) }
-                            max={ toPercent(MAX_RISK_LIMIT) }
-                            minorStepSize={ toPercent(0.001) }
-                            onBlur={ this.onBlur }
-                            stepSize={ toPercent(0.01) }
-                            value={ comparisonField }
-                            onValueChange={ this.onComparisonValueChange } />
-                    </label>
-                </div>
-            </div>
+        const ballotPollingFormField = (
```

Comment:
Didn't they want ballot polling audits eliminated from the UI? Or was that done somewhere else?

Reply:
Yes, that was removed. I'm holding on to the markup definition and (unobservable) form state in case we want it in the future. We just aren't rendering it right now.

Reply:
(removed here: 61c46a7e4c5776cbd0844003406240908d4b57ef)